app/helpers/application_helper.rb

```diff
@@ -174,4 +174,15 @@ def tab_matches(path)
     tab = tab_present(path)
     request.GET[:tab] == tab
   end
+
+  def app_javascript_include_tag(path = nil)
```

Comment:
`path` should be a required parameter as it is required for `Dir.glob`.

```suggestion
  def app_javascript_include_tag(path)
```